pkg/scheduler/api/node_info.go

```diff
@@ -76,20 +76,28 @@ func NewNodeInfo(node *v1.Node) *NodeInfo {
 }
 
 func (ni *NodeInfo) Clone() *NodeInfo {
-	res := NewNodeInfo(ni.Node)
+	newNode := NewNodeInfo(ni.Node)
 	glog.V(3).Infof("new node <%v>: capability %v,  allocatable %v, idle %v, used %v, backfilled %v, releasing %v", ni.Name, ni.Capability.MilliCPU,
 		ni.Allocatable.MilliCPU,
 		ni.Idle.MilliCPU,
 		ni.Used.MilliCPU,
 		ni.Backfilled.MilliCPU,
 		ni.Releasing.MilliCPU)
 
-	for _, p := range ni.Tasks {
-		glog.V(4).Infof("Adding task <%v/%v> to node <%v> with resource request %v", p.Namespace, p.Name, ni.Name, p.Resreq)
-		res.AddTask(p)
+	for _, task := range ni.Tasks {
+		glog.V(4).Infof("Adding task <%v/%v> to node <%v> with resource request %v", task.Namespace, task.Name, ni.Name, task.Resreq)
+		newNode.AddTask(task)
+	}
+
+	// set backfill status
+	for _, task := range newNode.Tasks {
+		task.IsBackfill = CheckBackfill(task.Pod)
```

Comment:
Can we move this logics into the for loop on line 87?

Reply:
same reason above. we can discuss in person.